src/backend/function/Fusion.Resources.Functions/Domains/Profile/ProfileSynchronizer.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Functions.Domains.Profile
```

Comment:
Do we need two files for this? Could not this be defined in the function?

Reply:
You are correct. When I started this I reckoned it would be some support files (models and whatnot). The trend of sub-classes eliminated this and ended up with 1 file. Refactored this, all in the function now.